bcrs-ui/src/App.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <div id="app">
```

Comment:
This `div` needs to be a `v-app` instead. Dunno why vue-cli scaffolded it this way!?
Ref: https://vuetifyjs.com/en/components/application

Reply:
Weird, let's make it right!